### PR TITLE
Updates to 7-15 revision data structures

### PR DESCRIPTION
**Pull request by evan-masseau**
Here is what I think the API payloads need to look like for upcoming 7-15 V3 API changes.
Note the PR's target branch: this is on top of this pending PR #91 
I tested these changes against prod on the VPN so that I could target 7-15.pre revision

### CODE REVIEW

File: Tests/KlaviyoSwiftTests/__Snapshots__/EncodableTests/testEventPayload.1.json [hunk 1]
```diff
@@ -2,14 +2,28 @@
...
+          "type" : "metric"
+        }
       },
       "profile" : {
-        "email" : "blob@email.com",
-        "location" : {
-          "city" : "blob city"
-        },
-        "stuff" : 2
+        "data" : {
```
Comment (evan-masseau) [inline Tests/KlaviyoSwiftTests/__Snapshots__/EncodableTests/testEventPayload.1.json:13]: oh that didnt go as planned

File: Sources/KlaviyoSwift/Version.swift [hunk 2]
```diff
@@ -7,5 +7,6 @@
 
 import Foundation
 
-public let __klaviyoSwiftName = "klaviyo-swift-sdk"
+// public let __klaviyoSwiftName = "klaviyo-swift-sdk"
```
Comment (evan-masseau) [inline Sources/KlaviyoSwift/Version.swift:10]: temporary until a fix is deployed to prod

File: Tests/KlaviyoSwiftTests/__Snapshots__/EncodableTests/testEventPayload.1.json [hunk 3]
```diff
@@ -2,14 +2,30 @@
...
+          "attributes" : {
+            "anonymous_id" : "anon-id",
+            "email" : "blob@email.com",
+            "properties" : {
+
+            }
+          },
+          "meta" : {
+            "identifiers" : {
```
Comment (jason-myers-klaviyo) [inline Tests/KlaviyoSwiftTests/__Snapshots__/EncodableTests/testEventPayload.1.json:22]: meta identifiers is deprecated no longer needed.  You only have to pas the top part now (attributes with identifiers in them, like you have above)

Comment (evan-masseau) [reply Tests/KlaviyoSwiftTests/__Snapshots__/EncodableTests/testEventPayload.1.json:22]: Oh weird I was just testing against the 7-15 alpha events API and it seemed to be necessary, will look again. 

(also damn that was a quick response, are you omniscient?)

Comment (jason-myers-klaviyo) [reply Tests/KlaviyoSwiftTests/__Snapshots__/EncodableTests/testEventPayload.1.json:22]: Ah, the change was made to /client/profiles but perhaps /client/events/ was not updated to match

I will investigate and see about removing it from there too.  In the meantime if it seems required, go ahead and leave it in here

Comment (jason-myers-klaviyo) [reply Tests/KlaviyoSwiftTests/__Snapshots__/EncodableTests/testEventPayload.1.json:22]: Oh, they are removing it but it is still open PR: https://github.com/klaviyo/app/pull/51575

The release is the same as yours though (7-15)

Comment (evan-masseau) [reply Tests/KlaviyoSwiftTests/__Snapshots__/EncodableTests/testEventPayload.1.json:22]: Awesome, I'm sure that means we shouldn't have our new push tokens API use meta identifiers either then. So glad you were watching this repo thanks!

**Pull request merged**